README.md

```diff
@@ -178,6 +181,77 @@ swap is initiated successfully, `loopd` will see the process through.
 
 To query in-flight swap statuses, run `loop monitor`.
 
+### Fees explained
+
+The following is an example output of a 0.01 BTC fast (non-batched) Loop Out
+swap from `testnet`:
+
+```bash
+$ loop out --amt 1000000 --fast
+Max swap fees for 1000000 sat Loop Out: 36046 sat
+Fast swap requested.
+CONTINUE SWAP? (y/n), expand fee detail (x): x
+
+Estimated on-chain sweep fee:        149 sat
+Max on-chain sweep fee:              14900 sat
+Max off-chain swap routing fee:      20010 sat
+Max off-chain prepay routing fee:    36 sat
+Max no show penalty (prepay):        1337 sat
+Max swap fee:                        1100 sat
+CONTINUE SWAP? (y/n):
+```
+
+Explanation:
+
+- **Max swap fees for <x> sat Loop Out** (36'046 sat): The absolute maximum in
```

Comment:
What is the `'` there? Another notation :)

Reply:
Ah, I forgot that other localizations use commas or other characters to separate thousands steps. Removed it.